base/math.jl

```diff
@@ -622,6 +622,8 @@ The article is available online at ArXiv at the link
 
 Compute the hypotenuse ``\\sqrt{\\sum |x_i|^2}`` avoiding overflow and underflow.
 
+See also [`LinearAlgebra.norm`](@ref).
```

Comment:
```suggestion
See also `norm` in the [`LinearAlgebra`](@ref man-linalg) standard library.
```
This is the original one, after 3 attempts to make the link work failed to please CI.